pkg/services/mssqldr/dbms_pair_registered_update.go

```diff
@@ -0,0 +1,40 @@
+package mssqldr
+
+import (
+	"context"
+
+	"github.com/Azure/open-service-broker-azure/pkg/service"
+)
+
+func (d *dbmsPairRegisteredManager) ValidateUpdatingParameters(
+	service.Instance,
+) error {
+	return nil
+}
+
+func (d *dbmsPairRegisteredManager) GetUpdater(
+	service.Plan,
+) (service.Updater, error) {
+	return service.NewUpdater(
+		service.NewUpdatingStep("updateAdministrators", d.updateAdministrators),
+		service.NewUpdatingStep("testPriConnection", d.testPriConnection),
```

Comment:
Here exists the same potential bug as I raised in #570 . The user may provide wrong password by mistake, which will cause step testConnection fails, but the wrong parameter will be written to store in step updateAdministrator and previous correct username and password will be overwritten.

Reply:
Yes, I almost forgot it. Thanks for raising it again.